AzurePullRequests.Restore/Enums/RestoreType.cs

```diff
@@ -1,10 +1,4 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
-namespace AzurePullRequests.Restore.Enums
+﻿namespace AzurePullRequests.Restore.Enums
```

Comment:
Interesting! Comment in file
![images](https://github.com/user-attachments/assets/d67b300b-efc4-4536-a340-80b2eddecf25)